process/slash/detector/roundHashCache_test.go

```diff
@@ -0,0 +1,141 @@
+package detector
+
+import (
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRoundDataCache_Add_OneRound_FourHeaders(t *testing.T) {
+	t.Parallel()
+
+	dataCache := NewRoundHashCache(1)
+
+	err := dataCache.Add(1, []byte("hash1"))
+	require.Nil(t, err)
+
+	err = dataCache.Add(1, []byte("hash1"))
+	require.Equal(t, process.ErrHeadersNotDifferentHashes, err)
+
+	err = dataCache.Add(1, []byte("hash2"))
+	require.Nil(t, err)
+
+	err = dataCache.Add(1, []byte("hash3"))
+	require.Nil(t, err)
+
+	require.Len(t, dataCache.cache, 1)
+	require.Len(t, dataCache.cache[1], 3)
+
+	require.Equal(t, []byte("hash1"), dataCache.cache[1][0])
```

Comment:
can you remove empty lines between consecutive requires?

Reply:
Yes, removed them